src/main/java/org/junit/runners/MethodSorters.java

```diff
@@ -25,6 +26,13 @@
     JVM(null),
 
     /**
+     * Sorts the test methods in pseudo-random order.
+     * This sort option generates and displays (to stdout) a
+     * seed value for the random number generator.
+     */
+    RANDOM(null),
```

Comment:
BTW, if you want the ordering provided by `RANDOM` to be consistent when you pass the same seed, then you'll need to actually order the methods in a deterministic way before you do the shuffling.